lib/solr_wrapper/instance.rb

```diff
@@ -101,37 +101,41 @@ def restart
     end
 
     ##
-    # Check the status of a managed Solr service
-    def status
-      return true unless config.managed?
+    # Is Solr running?
+    def started?
+      status_info =~ /running on port #{port}/ && true || false
```

Comment:
OK, deferring to your preference on this.  Removing it.  (Aside, I'm curious and open to using an explicit `match()` way of doing as an alternative coding style; rubocop doesn't approve of `!!`)